docs/guide/008-technical-document.adoc

```diff
@@ -1,4 +1,4 @@
 == Technical Document
 
-More details on the WCMP metadata can be found at http://wis.wmo.int/MD_Index.
+More details on the WCMP metadata can be found at https://community.wmo.int/activity-areas/wis/wcmp.
```

Comment:
@amilan17 please check link - thx!

Reply:
I think we should delete this section. We already have a link at the beginning to this URL.

Reply:
Well, this is the only sentence in the "008-technical-document.adoc". We can remove the whole document (chapter then).

Reply:
removed chapter